crossdock/client/tchclient/thrift.go

```diff
@@ -39,21 +44,377 @@ func runThrift(t crossdock.T, call call) {
 	}
 	token := random.String(5)
 
+	call.Channel.Peers().Add(call.ServerHostPort)
```

Comment:
optional: instead of calling `Peers().Add` here, you can instead pass an option to `thrift.NewClient`,
```go
thrift.NewClient(call.Channel, serverName, &thrift.CallOptions{HostPort: hostPort})
```